src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -64,6 +64,7 @@ public CommandResult execute(Model model) throws CommandException {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof AddCommand // instanceof handles nulls
-                && toAdd.equals(((AddCommand) other).toAdd));
+                && ((toAdd.getName().equals(((AddCommand) other).toAdd.getName()))
```

Comment:
@tiewweijian just a quick clarification on this! Was wondering where the equal method for `AddCommand` is utilised, and if we should only check for the same `studentId`?

Hmm because I think the `hasSameCandidate()` method only checks for candidates with the same `studentId`, so was curious about the need for checking for name as well! :)

Reply:
Hey @charmainehly ! I added this one in because it affected one of the test cases 
`assertFalse(addAliceCommand.equals(addBobCommand));` in `AddCommandTest`, both Alice and Bob had the same ID, and so the outcome of the `equals` statement became true, and the assertion gave an error. 

Not sure why it passed the test case last time (or not sure if I accidentally messed with it), but thought of adding the `getName` as a comparator for a tighter comparison. (But not too tight yet too) 

Hmm do you think I should revert it back to just comparing their IDs?

Reply:
Hmmmm, might be minor, but IMO was thinking we should just check for the same `studentId`, and maybe switch the test case to `assertTrue`? 

Was thinking that technically even if the names are different, so long as the `studentId` is the same they are still considered different candidates - I think the test case might be returning something different because we didn't include `withStudentId("...")` in the `CandidateBuilder` for the `AddCommandTest` equals test?

Reply:
Mm perhaps I could switch up the test case ! 
I didn't wanna touch the test case initially because I was afraid it will mess with other functions, but let me see how I could change it ! Perhaps adding a `.withStudentId()` like you mention will help!

Reply:
@tiewweijian will deleting the `getName` check in the equals method affect any other test cases? if not we can just delete the `getName check` :O